.travis.yml

```diff
@@ -19,18 +19,18 @@ before_script:
 script: make
 
 after_success: 
-    - mkdir -p render_output
+    - make doxygen-doc
     - make bundle
-    #- ./radiance_cli -b nyancat.gif -o render_output
+    #- ./radiance_cli -a
 
```

Comment:
Should we make a cronjob on radiance.video to do this?

Reply:
Yeah, it needs a GUI though :\